docs/quickstart.rst

```diff
@@ -2,54 +2,17 @@
 Quick start
 =============
 
-Example locustfile.py
-=====================
+In Locust you define your user behaviour in plain Python code. You then use the ``locust`` command and (optionally) its web interface to spawn and simulate a number of those users while gathering request statistics.
 
-When using Locust you define the behaviour of users in Python code, and then you have the ability to 
-simulate any number of those users while gathering request statistic. The entrypoint for defining the 
-user behaviour is the `locustfile.py`.
-
-.. note::
-
-    The ``locustfile.py`` is a normal Python file that will get imported by Locust. Within it you 
-    can import modules just as you would in any python code.
-    
-    The file can be named something else and specified with the `-f` flag to the ``locust`` command.
-
```

Comment:
I think the note about the locustfile.py being a normal Python file, that gets imported by Locust, and where one can import other python modules, is important. I've seen many examples where people seem to haven't realised that they can split their load test suite across multiple modules.

Reply:
I think that should be covered by "In Locust you define your user behaviour in plain Python code." at the top, but if you dont agree, you can add it back in where you think is appropriate.

Reply:
True, it should, but I think it can be worth making it super clear. That was the reason I added it in IIRC (I think the text about locustfiles being normal python files were already there when the note was added).

Reply:
re-added it further down